aqt/combinations.json

```diff
@@ -1,6 +1,9 @@
 [{"qt": [
   {"os_name": "linux",   "target": "desktop", "arch": "gcc_64"},
   {"os_name": "linux",   "target": "desktop", "arch": "wasm_32"},
+  {"os_name": "linux",   "target": "android", "arch": "android"},
+  {"os_name": "linux",   "target": "android", "arch": "android_x86_64"},
+  {"os_name": "linux",   "target": "android", "arch": "android_arm64_v8a"},
```

Comment:
Is it any way to specify version range in the database? If we can do it, we can check it.
For example, "version_maximum": "5.13", "version_minimum":"5.14"

It can be also useful for WASM module.

Reply:
Yes, this is possible. However, I have to admit I don't think this database is a good idea at all. Let the user do whatever combinations they like and print an error if a combination does not exist. Or at least add a parameter to make the database check a warning instead of an error, so one can test combinations in alphas of Qt etc. that are not covered by the database yet.

Reply:
OK, i'd like to merge it as is.